flake-module.nix

```diff
@@ -95,14 +89,22 @@ ihpFlake:
                 ihp = ihp;
                 haskellPackagesDir = cfg.projectPath + "/Config/nix/haskell-packages";
             };
+            allHaskellPackages = p: [
+                p.cabal-install
+                p.base
+                p.wai
+                p.text
+                p.hlint
+                p.ihp
+            ] ++ (cfg.haskellPackages p);
```

Comment:
I'm not sure about this change, this feels like a kind of breaking change in how the IHP nix interface works

Reply:
Okay, yeah, makes sense, will roll back